vscode/webviews/Chat.tsx

```diff
@@ -322,6 +344,14 @@ export const Chat: React.FunctionComponent<React.PropsWithChildren<ChatboxProps>
                         eventName: 'CodyVSCodeExtension:chatActions:editLast:executed',
                         properties: { source: 'chat', eventType: 'keyDown' },
                     })
+                    postMessage?.({
+                        command: 'recordEvent',
+                        feature: 'chatActions:editLast',
+                        action: 'executed',
```

Comment:
This shouldn't use `postMessage`, it should use `telemetryRecorder`, but naming needs adjusting too:

```suggestion
                        feature: 'cody.chatActions',
                        action: 'editLast',
```

Reply:
updated! Thank you 🙇